added_mass/proposal.md

```diff
@@ -0,0 +1,304 @@
+<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>
+
+# Fluid Added Mass Proposal
+
+* **Authors**:
+Louise Poubel `<louise@openrobotics.org>`,
+Andrew Hamilton `<hamilton@mbari.org>`,
+Michael Anderson `<anderson@mbari.org>`
+* **Status**: Draft
+* **SDFormat Version**: 1.10
+* **`libSDFormat` Version**: 13.X (Gazebo Garden)
+
+## Introduction
+
+This proposal suggests the addition of elements under the `//link/inertial` element
+to support the inclusion of [fluid added mass] (https://en.wikipedia.org/wiki/Added_mass)
+effects in SDFormat 1.10 and support for parsing it in libSDFormat 13.
+
+Current implementations, such as
+[gz::sim::systems::Hydrodynamics](https://gazebosim.org/api/gazebo/6.9/classignition_1_1gazebo_1_1systems_1_1Hydrodynamics.html#details),
+include forces due to added mass effects by estimating these forces from the last time-steps
+acceleration value and applying these estimates as separate external forces, reducing simulation accuracy and stability.
+
+Native support of added mass through the SDF specification makes it possible for physics
+engines to handle these terms together with other inertial terms during the physics update
+step.
+
+## Document summary
+
+The proposal includes the following sections:
+
+* [Motivation](#motivation): An explanation of the background and rationale behind the proposal
+* [Proposed implementation](#proposed-implementation): Terms to be added to the SDFormat specification and C++ API
+
+## Motivation
+
+Newton's second law describes how forces affect a body's motion in an inertial frame. It can be written as:
+
+$$ (\bf{M} + \bf{\mu})   \ddot{\bf x} = \sum{F({\bf x}, t)} $$
```

Comment:
Note that this equation is only valid for translation; for the rotation of a rigid body, this becomes [Euler's equations](https://en.wikipedia.org/wiki/Euler%27s_equations_(rigid_body_dynamics)); and in the presence of cross-terms in the spatial inertia matrix, this becomes equation (2.90) from [1].